qcodes/dataset/sqlite/query_helpers.py

```diff
@@ -60,19 +61,29 @@ def many(curr: sqlite3.Cursor, *columns: str) -> list[Any]:
         return [res[0][c] for c in columns]
 
 
-def many_many(curr: sqlite3.Cursor, *columns: str) -> list[list[Any]]:
+def many_many(curr: sqlite3.Cursor,
+              columns: str | List[str] | Tuple[str],
```

Comment:
Thanks, I will change the typing.

For the other question of the API, I tried to used the old code as much as possible, hence the modification of `many` and `many_many`.
I guess it can be done another way.